cmd/net.go

```diff
@@ -77,18 +77,58 @@ func getHostIP4(host string) (ipList set.StringSet, err error) {
 	return ipList, err
 }
 
+// byLastOctetValue implements sort.Interface used in sorting a list
+// of ip address by their last octet value.
+type byLastOctetValue []net.IP
```

Comment:
Will define explicitly @donatello. 

@balamurugana That is incorrect. A string based mechanism is not what we want here. All we want is a highest octet on the system to be the first in the element not just by string comparing and moving localhost to the end.